src/app/screens/auth/components/shopping-cart/shopping-cart.component.ts

```diff
@@ -0,0 +1,17 @@
+import { Component, Input } from '@angular/core';
+import { Book } from '../../interfaces/book.interface';
+
+@Component({
+  selector: 'app-shopping-cart',
+  templateUrl: './shopping-cart.component.html',
+  styleUrls: ['./shopping-cart.component.scss']
+})
+export class ShoppingCartComponent {
+
+  @Input() data: Array<Book> = [];
+
+  removeBook(book: Book) {
+    this.data.splice(this.data.findIndex(x => x === book), 1);
```

Comment:
Use filter array method for more clean code.
`this.data = this.data.filter(x => x !== book);`